namespace-package/plugins/data-repo-plugs/data_repo/read.py

```diff
@@ -0,0 +1,43 @@
+from importlib import resources, import_module
+
+
+def collect():
+    print("Collecting readers")
+    readers = {}
+    for item in resources.files(f"{__package__}.readers").iterdir():
+        if item.stem == "__pycache__":
+            continue
+        try:
+            read_function = import_module(
+                f".{item.stem}", f"{__package__}.readers"
+            ).read
+            if callable(read_function):
+                readers[item.stem] = read_function
+        except ImportError:
+            continue
+        except AttributeError as e:
```

Comment:
This `e` isn't used and can be removed: `except AttributeError:`

Reply:
Updated here and in the article